### PR TITLE
Fix latex error summary

### PR DESCRIPTION
**Pull request by kilacoda**
## Changelog / Overview

Fix f-string formatting introduced by #1564, which earlier produced a hardcoded value instead of the error summary 

## Motivation

Apparently, #1564 added extra log info for LaTeX errors, but for some reason, this line was added which prints `LaTeX compilation error: {log[error_pos][2:])}` instead of the actual error as I assume was intended:
https://github.com/ManimCommunity/manim/blob/e85599e9829460948ac63f620a9bc737e8b7a5c7/manim/utils/tex_file_writing.py#L198

With my change, it looks like this now:
image

## Explanation for Changes

Uses the first element of the `error_pos` list and adds the `f` modifier to the start of the string.

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Testing Status

```py
class TexErrorAttempt(Scene):
    def construct(self):
        self.add(Tex("\\int_x^x"))
```

image

## Further Comments

Alternatively, this could have been done by setting `error_pos` to the 0th index of its current value.

## Checklist
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [ ] I have written a descriptive PR title (see top of PR template for examples)
- [ ] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by clarfonthey** [comment]
Thank you -- this was an issue that came up in rebasing that wasn't noticed.

**Comment by kilacoda** [comment]
@clarfonthey , just to clarify, this was the intended use of the `error_pos` list right? Is it always supposed to only have one integer value as its contents?

**Comment by clarfonthey** [comment]
Actually, no; it should be inside the loop.

**Review by Nikhil0504 — APPROVED** [review]

File: manim/utils/tex_file_writing.py [hunk 1]
```diff
@@ -195,7 +195,9 @@ def compile_tex(tex_file, tex_compiler, output_format):
                 if error_pos:
                     with open(tex_file, "r") as g:
                         tex = g.readlines()
-                        logger.error("LaTeX compilation error: {log[error_pos][2:])}")
+                        logger.error(
+                            f"LaTeX compilation error: {log[error_pos[0]][2:]}"
+                        )
                         for log_index in error_pos:
```
Comment (kilacoda) [inline manim/utils/tex_file_writing.py:201]:
```suggestion
                        for log_index in error_pos:
                            logger.error(
                                f"LaTeX compilation error: {log[log_index][2:]}"
                            )
```

@clarfonthey like this?

Comment (clarfonthey) [reply manim/utils/tex_file_writing.py:201]: Yes

**Pull request merged**